tarantool/connection.py

```diff
@@ -81,6 +81,7 @@
     InternalError,
```

Comment:
https://github.com/tarantool/tarantool-python/blob/70b0a3debe5ffa7f1c0b5c84d6fb9e28cbe26152/tarantool/connection.py#L96-L241

I think that crud native API should become the part of `Connection` API.

Reply:
Done, added